websites/B/Brain.fm/presence.ts

```diff
@@ -67,7 +69,7 @@ presence.on("UpdateData", () => {
 		else if (["/focus", "/relax", "/sleep"].includes(pathname)) {
 			const activity = document
 				.querySelector(
-					"#root > div > div > div.sc-sCrnh.fGBZrg > div.sc-gJLCiq.iGquue > div.sc-kBizdN.gGaVww > div.sc-jNpQCG.dAEozB > div > div > span"
+					"#root > div.sc-inJWWq.lbdOVj > div > div.sc-bImIVK.gtfUIb > div.sc-dZpgSj.bzGlhX > div.sc-jDIYFk.fbXWvM > div.sc-bdytox.kExulF > div > div > span"
 				)
```

Comment:
@theusaf how about the following instead?

It works with XPath :)
```ts
document.evaluate('//span[text()="Activity"]', document)?.iterateNext()?.parentElement?.querySelectorAll("span")?.[1]?.textContent
```

Reply:
Seems interesting, this could prevent future breakage from class names changing.

Reply:
I think it's a good idea.

Reply:
I managed to clean up some more searches by using their 'test-id' attributes. 
Personally I'm not really experienced with using XPath, so I'm using regular querySelectors for now.